src/hooks/useReportScrollManager/types.native.ts

```diff
@@ -0,0 +1,10 @@
+import {ActionListContextType} from '../../pages/home/ReportScreenContext';
```

Comment:
You don't need two types.ts files, the point is to have one and create common types that work for all platform specific implementations at once. This way we make sure these functions are compatible across platforms.

Reply:
So remove types.native.ts and use just types.ts 😄

Reply:
Updated, let me know if that's what you meant 😄